custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
```

Comment:
Consider using template literals for string interpolation instead of faker functions for static text values. This improves code readability and maintainability.